utils/utils.go

```diff
@@ -0,0 +1,49 @@
+package utils
```

Comment:
please use meaningful names for packages.

Reply:
I'm using the already defined one i.e [github.com/src-d/metadata-retrieval/utils/ctxlog](https://github.com/src-d/metadata-retrieval/blob/master/github/downloader.go#L11)

Reply:
as you can see log utils have a meaningful name `ctxlog`. It says it is not a random util but log that works with context.

Reply:
shouldn't be `utils` the parent of `utils/ctxlog`?

Reply:
`utils` exists only to avoid polluting root of the project.

According to the content of the file something like
```
httpbody.CloneResponse(resp *http.Response) ([]byte, error)
httpbody.CloneRequest(req *http.Request) ([]byte, error)
```
should be fine.

Reply:
That's a pretty good suggestion indeed.
I understood that we were talking about the package name, so I didn't get your point. Moving that helpers into its own package is way better!.